amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/DynamoUtilsTest.java

```diff
@@ -0,0 +1,58 @@
+package software.amazon.kinesis.leases;
+
+import org.junit.Assert;
+import org.junit.Test;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.ArrayList;
```

Comment:
I think imports of the same type should be arranged alphabetically ie. ArrayList before Collection and HashMap.